modules/eco-poison-pill-about-watchdog.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/eco-poison-pill-operator.adoc
+
+:_content-type: CONCEPT
+[id="about-watchdog-devices_{context}"]
+= About watchdog devices
+
+Watchdog devices are of two kinds:
+* Hardware watchdog 
+* Software watchdog (or softdog)
+
+Hardware watchdog and softdog devices have electronic or software timers, respectively. These watchdog devices are used to provide fencing. When the cluster is in a healthy state, it regularly restarts the watchdog timer at set intervals. This timer might elapse due to fault conditions such as CPU starvation or unhealthy nodes. The watchdog or softdog device now assumes that a fault has occurred and the device triggers a forced reset of the node.
+
+Hardware watchdog devices are more reliable than softdog devices.
+
+[id="understanding-pp-watchdog_{context}"]
+== Understanding Poison Pill Operator behavior with watchdog devices
+
+The Poison Pill Operator uses these watchdog devices to determine the remediation strategy. Since hardware watchdog devices are more reliable than softdog devices, the Operator prefers using the former first.
```

Comment:
```suggestion
The Poison Pill Operator determines the remediation strategy based on the watchdog devices present. Since hardware watchdog devices are more reliable than softdog devices, the Operator will prefer the former.
```